src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -16,6 +15,9 @@
 
 namespace NzbDrone.Core.Download.Clients.Deluge
 {
+    // this is here to resolve ambiguity in GetValueOrDefault extension method in net core 3
+    using NzbDrone.Common.Extensions;
```

Comment:
Should we just rename instead of having to remember this fix everywhere? What’s the behavior difference between the two that are ambiguous?